doc/user/content/demos/business-intelligence.md

```diff
@@ -199,9 +201,9 @@ Materialize to maintain for us.
     SELECT * FROM query01;
     ```
 
-    If you run this query multiple times, you should see the results change, and
-    the answers should come back pretty quickly. (How quickly depends on the
-    speed of your computer, but it should be some small fraction of a second.)
+    If you run this query multiple times, you should see the results change,
+    and the answers should come back pretty quickly. (How quickly depends on
+    the speed of your computer, but it should be in milliseconds.)
```

Comment:
Maybe “but several milliseconds is typical” to avoid making a strong claim about our speed on everyone’s machine?

Reply:
updated to `but sub-second responses are typical.`